schemas/public/ekscluster-kfd-v1alpha2.json

```
@@ -1266,27 +1316,28 @@
     "Spec.Distribution.Modules.Ingress.CertManager.ClusterIssuer": {
       "type": "object",
       "additionalProperties": false,
+      "description": "Configuration for the cert-manager's ACME clusterIssuer used to request certificates from Let's Encrypt.",
       "properties": {
         "name": {
           "type": "string",
-          "description": "The name of the cluster issuer"
+          "description": "Name of the clusterIssuer."
         },
         "email": {
           "type": "string",
           "format": "email",
-          "description": "The email of the cluster issuer"
+          "description": "The email address to use during the certificate issuing process."
         },
         "type": {
           "type": "string",
           "enum": [
             "dns01",
             "http01"
           ],
-          "description": "The type of the cluster issuer, must be ***dns01*** or ***http01***"
+          "description": "The type of the clusterIssuer, must be `dns01` for using DNS challenge or `http01` for using HTTP challenge."
         },
         "solvers": {
           "type": "array",
-          "description": "The custom solvers configurations"
+          "description": "List of challenge solvers to use instead of the default one for the `http01` challenge."
```

```suggestion
          "description": "The list of challenge solvers to use instead of the default one for the `http01` challenge. Check [cert manager's documentation](https://cert-manager.io/docs/configuration/acme/#adding-multiple-solver-types) for examples for this field."
```
Just to keep the same wording as the other fields. Also, add a link to cert manager docs for the syntax to be used here